packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/create-model-customization-job.ts

```diff
@@ -0,0 +1,396 @@
+import { Construct } from 'constructs';
+import * as bedrock from '../../../aws-bedrock';
+import * as ec2 from '../../../aws-ec2';
+import * as iam from '../../../aws-iam';
+import * as kms from '../../../aws-kms';
+import * as sfn from '../../../aws-stepfunctions';
+import { Aws, Stack, Token } from '../../../core';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * The customization type.
+ *
+ * @see https://docs.aws.amazon.com/bedrock/latest/userguide/custom-models.html
+ */
+export enum CustomizationType {
+  /**
+   * Fine-tuning
+   *
+   * Provide labeled data in order to train a model to improve performance on specific tasks.
+   */
+  FINE_TUNING = 'FINE_TUNING',
+  /**
+   * Continued pre-training
+   *
+   * Provide unlabeled data to pre-train a foundation model by familiarizing it with certain types of inputs.
+   */
+  CONTINUED_PRE_TRAINING = 'CONTINUED_PRE_TRAINING',
+}
+
+/**
+ * The key/value pair for a tag.
+ */
+export interface ITag {
+  /**
+   * Key for the tag.
+   */
+  readonly key: string;
+  /**
+   * Value for the tag.
+   */
+  readonly value: string;
+}
+
+/**
+ * VPC configuration
+ */
+export interface IBedrockCreateModelCustomizationJobVpcConfig {
+  /**
+   * VPC configuration security groups
+   */
+  readonly securityGroups: ec2.ISecurityGroup[];
+  /**
+   * VPC configuration subnets
+   */
+  readonly subnets: ec2.ISubnet[];
+}
+
+/**
+ * Properties for invoking a Bedrock Model
+ */
+export interface BedrockCreateModelCustomizationJobProps extends sfn.TaskStateBaseProps {
+  /**
+   * The base model.
+   */
+  readonly baseModel: bedrock.IModel;
+  /**
+   * A unique, case-sensitive identifier to ensure that the API request completes no more than one time.
+   * If this token matches a previous request, Amazon Bedrock ignores the request, but does not return an error.
+   *
+   * @see https://docs.aws.amazon.com/AWSEC2/latest/APIReference/Run_Instance_Idempotency.html
+   *
+   * @default - no client request token
+   */
+  readonly clientRequestToken?: string;
+  /**
+   * The customization type.
+   *
+   * @default FINE_TUNING
+   */
+  readonly customizationType?: CustomizationType;
+  /**
+   * The custom model is encrypted at rest using this key.
+   *
+   * @default - no encryption
+   */
+  readonly kmsKey?: kms.IKey;
+  /**
+   * A name for the resulting custom model.
+   */
+  readonly customModelName: string;
+  /**
+   * Tags to attach to the resulting custom model.
+   *
+   * @default - no tags
```

Comment:
```suggestion
   * Tags to attach to the resulting custom model.
   *
   * The maximum number of tags is 200.
   *
   * @default - no tags
```
Can you please add constraints documentation for validated props?